### DISCUSSION TITLE
It would be great to have Milestones that are not pinned to a repo

### DISCUSSION THREAD

**Discussion by olibates** [community/community#7405] [Projects and Issues]
Using the new projects feature. It's great! Small pain is that we'd like to define milestones to track work against but they're repo-specific even though our actual milestones are not. Right now we have to duplicate them which is a pain, but it would be great if milestones could be multi-rep somehow.

**Comment by dpcleitao** [comment]
100% with this. We are using a specific Single Select field to achieve this right now, but it is not the right solution.

**Comment by ghost** [comment]
Duplicate of https://github.com/github-community/community/discussions/6296 (as this has more up-votes)
